ci/deploy-gh-pages.sh

```diff
@@ -13,6 +13,10 @@ set -x -e
 if [[ "$GITHUB_EVENT_NAME" == "release" ]]; then
     # Get the tag name without the "refs/tags/" part
     version="${GITHUB_REF#refs/*/}"
+    # Truncate the version to 6.x
+    if [[ "${GITHUB_REF#refs/*/}" =~ ^6\.[0-9]+.[0-9]+ ]]; then
```

Comment:
Two suggestions here:

1. Change the leading `6` to `6-9` so that the script is expected to work for GMT 6 to GMT 9.
2. Change the 2nd `.` to `\.` to match a literal dot `.`.
```suggestion
    # Truncate the version to X.Y
    if [[ "${GITHUB_REF#refs/*/}" =~ ^[6-9]\.[0-9]+\.[0-9]+ ]]; then
```